### PR TITLE
docs: remove duplicate import from @angular/core

### PR DESCRIPTION
**Pull request by xiaohanxu-nick**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

The [get start example](https://angular.io/start#pass-data-to-a-parent-component) code can be simplified, now it looks like:
```
import { Component } from '@angular/core';
import { Input } from '@angular/core';
import { Output, EventEmitter } from '@angular/core';
import { Product } from '../products';
```
it violate [no-duplicate-imports](https://eslint.org/docs/rules/no-duplicate-imports) rule from ESLint.
Issue Number: N/A

## What is the new behavior?
Could merge the two import of `@angular/core` in to a simple one, which is more clearer.

```
import { Component, Input, Output, EventEmitter} from '@angular/core';
import { Product } from '../products';
```
## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
Thanks for suggesting this change @xiaohanxu-nick.
I think looking at the rest of the guide and example this is a bit more tricky.

First, this change does not deduplicate all the `@angular/core` imports from this file, since we also import `Component` here too.

Grouping all these together is a bit awkward because elsewhere in the guide it references the `product-alerts.component.1.ts` file, which needs to have doc-regions for the code "as 
 it is generated" by the CLI, and also after we have imported the initial `Input` decorator.

I think the full solution is to use a bit of a hack. In `product-alerts.component.1.ts` we could change it to look like:

```ts
// #docplaster
/*
// #docregion as-generated
import { Component, OnInit } from '@angular/core';
// #enddocregion as-generated
*/
// #docregion as-generated, imports
import { Component, OnInit, Input } from '@angular/core';
import { Product } from '../products';
// #enddocregion imports
```

This allows the "as-generated" doc-region to be captured without it actually affecting the code (since it is commented out).
Then in the `product-alerts.component.ts` file we can change it to:

```ts
// #docregion imports
import { Component, Input, Output, EventEmitter } from '@angular/core';
import { Product } from '../products';
// #enddocregion imports
```

Can you try making these changes?

**Comment by xiaohanxu-nick** [comment]
> Thanks for suggesting this change @xiaohanxu-nick.
> I think looking at the rest of the guide and example this is a bit more tricky.
> 
> First, this change does not deduplicate all the `@angular/core` imports from this file, since we also import `Component` here too.
> 
> Grouping all these together is a bit awkward because elsewhere in the guide it references the `product-alerts.component.1.ts` file, which needs to have doc-regions for the code "as
> it is generated" by the CLI, and also after we have imported the initial `Input` decorator.
> 
> I think the full solution is to use a bit of a hack. In `product-alerts.component.1.ts` we could change it to look like:
> 
> ```ts
> // #docplaster
> /*
> // #docregion as-generated
> import { Component, OnInit } from '@angular/core';
> // #enddocregion as-generated
> */
> // #docregion as-generated, imports
> import { Component, OnInit, Input } from '@angular/core';
> import { Product } from '../products';
> // #enddocregion imports
> ```
> 
> This allows the "as-generated" doc-region to be captured without it actually affecting the code (since it is commented out).
> Then in the `product-alerts.component.ts` file we can change it to:
> 
> ```ts
> // #docregion imports
> import { Component, Input, Output, EventEmitter } from '@angular/core';
> import { Product } from '../products';
> // #enddocregion imports
> ```
> 
> Can you try making these changes?

Thanks for your suggestions. I have updated accordingly.

File: aio/content/examples/getting-started/src/app/product-alerts/product-alerts.component.1.ts [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+/*
+// #docregion as-generated
 import { Component, OnInit } from '@angular/core';
 // #enddocregion as-generated
-import { Input } from '@angular/core';
+*/
+// #docregion as-generated, imports
+import { Component, OnInit, Input } from '@angular/core';
 import { Product } from '../products';
 // #enddocregion imports
```
Comment (petebacondarwin) [inline aio/content/examples/getting-started/src/app/product-alerts/product-alerts.component.1.ts:10]: Oops my mistake. This should be:

```suggestion
// #docregion imports
import { Component, OnInit, Input } from '@angular/core';
import { Product } from '../products';
// #enddocregion imports
// #docregion as-generated
```

Comment (xiaohanxu-nick) [reply aio/content/examples/getting-started/src/app/product-alerts/product-alerts.component.1.ts]: updated. Thanks.

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Comment by petebacondarwin** [comment]
Marvellous thanks!

**Pull request closed without merging**